.travis.yml

```diff
@@ -24,6 +30,7 @@ cache:
 install:
     - mkdir -p build/logs
     - mv ${HOME}/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini ${HOME}/xdebug.ini || return 0
+    - 'if [ "$SYMFONY_VERSION" != "" ]; then sed -i "s/\"symfony\/\([^\"]*\)\": \"^2[^\"]*\"/\"symfony\/\1\": \"$SYMFONY_VERSION\"/g" composer.json; fi'
```

Comment:
this is 1-1 took from PHP CS Fixer, right ?

Reply:
Yes, is it a problem?

Reply:
no, taking a confirmation that I don't need to analyse the line ;)